### ISSUE TITLE
Issue 6: DataLoader

### ISSUE THREAD

**Issue by DannyPhongcoderso1** [DannyPhongcoderso1/DL-retinal-vessel-segmentation-final#6]
- [ ] Tạo:
    train_loader
    val_loader
    test_loader

- [ ] batch_size:
    4 / 8

- [ ] shuffle=True cho train

- [ ] Debug:
    print shape

Expected:
    image: [B,3,256,256]
    mask:  [B,1,256,256]
